### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by EmilienM**
Just add some docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,86 @@
...
+       --github-cloner-key /path/to/cloner.key --run-make
+   ```
+
+## Contributing
+
+Contributions are welcome! Please submit a pull request or open an issue to discuss any changes or features.
+
+---
+
+For any questions or support, feel free to contact the maintainers.
```
Comment (pierreprinetti) [inline README.md]: how? Maybe using Github issues?

**Comment by pierreprinetti** [comment]
Nice, thanks! I'll put a /hold go give you the chance to address the nit if you so want. Otherwise, just go ahead and merge :)

/lgtm
/hold

**Review by mandre — COMMENTED** [review]
/lgtm

**Pull request merged**